running/architecture.rst

```diff
@@ -2,3 +2,72 @@ Architecture
 ============
 
 Deconst is distributed as a set of Docker containers, deployed by an Ansible playbook.
+
+Terminology
+-----------
+
+.. glossary::
+
+  content repository
+    Version controlled repository containing markup that provides content for some subset of the
+    completed site. To begin, we'll assume that all of these are in ``git`` repositories on
+    `GitHub <github.com>`_. Most of our documentation will be in Sphinx reStructuredText, but the
+    developer blog will be in Markdown as part of a Jekyll site.
```

Comment:
Well, we could *theoretically* support "content" from just about anywhere if we write enough code to do it. I assume your point is to make it clear that we aren't going to couple ourselves to GitHub? I can rephrase a little, sure.

> ...it's entirely plausible that other content might be sourced from a database or other storage mechanism...

Wait, is someone actually doing this? Is supporting that a requirement? I don't think it would be *horribly* difficult to support that (we'd just have to poll it with a preparer from a cron job, probably) but that seems wildly inconvenient for everyone involved.